src/main/java/org/candlepin/resource/EnvironmentResource.java

```diff
@@ -354,7 +354,7 @@ public AsyncJobStatusDTO demoteContent(@Verify(Environment.class) String envId,
     @SecurityHole(activationKey = true)
     @Transactional
     public ConsumerDTO createConsumerInEnvironment(String envId, ConsumerDTO consumer,
-        String userName, String activationKeys) throws BadRequestException {
+        String userName, String activationKeys, String owner) throws BadRequestException {
```

Comment:
The new input doesn't appear to be used anywhere outside of one test. What is the intent for this?

Reply:
I wasn't sure about this one. Just using our Request/Response would have been enough, but the owner here is necessary for activation key auth so I thought it might be better to have it in the API explicitly.

Reply:
Looking at this: https://spec.openapis.org/oas/latest.html#securitySchemeObject

Reply:
Done.